packages/web/components/transactions/transaction-details.tsx

```diff
@@ -1,21 +1,42 @@
 import { Transition } from "@headlessui/react";
+import { getShortAddress } from "@osmosis-labs/utils";
 import classNames from "classnames";
+import dayjs from "dayjs";
 import Image from "next/image";
 import { FunctionComponent } from "react";
 
 import { Icon } from "~/components/assets";
+import { FallbackImg } from "~/components/assets";
 import { CopyIconButton } from "~/components/buttons/copy-icon-button";
 import IconButton from "~/components/buttons/icon-button";
 import { Button } from "~/components/ui/button";
 import { ModalBase, ModalBaseProps } from "~/modals/base";
+import { formatPretty } from "~/utils/formatter";
+
+import { FormattedTransaction } from "../../../server/src/queries/complex/transactions/transaction-types";
```

Comment:
I think we could import this from `@osmosis-labs/server` instead of the large relative path